### PR TITLE
Language change tables to datasets and copy edits

### PR DESCRIPTION
**Pull request by brittanymicek**
hey sorry. somehow this got buried. reviewed and requested merge of my pr! @andrewxhill @iriberri #244

### CODE REVIEW

File: faqs.md [hunk 1]
```diff
@@ -63,27 +63,25 @@ CartoDB understands and utilizes the geographic coordinates written as [decimal
...
+ First and foremost, a database table has a lot more storage considerations than a CSV. For example, indexes. It is also important to consider that the actual way that data is stored on disks is optimized for lookup and retrieval speed over storage space. A CSV is made so you can optimally store data - you open it, load it all into a program for a limited amount of time while you edit it, then save it, and are done. The database has things like data types, etc. 
 
-### Can I synchronize my tables in real-time?
+### Can I synchronize my datasets in real time?
 
 <p class="wrap-border"><img src="{{ '/img/layout/faqs/force-sync.png' | prepend: site.baseurl }}" alt="Can I synchronize my tables in real time" /></p>
 
-By using the feature of sync tables, the shortest automatic syncing interval is 1 hour, but you can force a manual synchronization each 15 minutes.
+By using the sync dataset feature, the shortest automatic syncing interval is 1 hour, but you can force a manual synchronization each 15 minutes.
```
Comment (iriberri) [inline faqs.md]: We maintain here "Sync tables" as it's like the brand of a feature :)

File: faqs.md [hunk 2]
```diff
@@ -63,27 +63,25 @@ CartoDB understands and utilizes the geographic coordinates written as [decimal
...
 
-### I have a sync table, how can I modify the column types?
+### I have a sync dataset, how can I modify the column types?
```
Comment (iriberri) [inline faqs.md]: Same than above, we maintain "Sync table"

File: faqs.md [hunk 3]
```diff
@@ -351,15 +349,15 @@ cartodb.createVis('map', 'http://documentation.cartodb.com/api/v2/viz/2b13c956-e
...
 
-## Your account
+## Your Account
 
 ### What are the geocoding credits?
 
 The geocoding credits are the quota for performing geocodings. With our geoocoding service you can translate addresses into coordinates for points that will be shown in your map. Each geocoding wastes one geocoding credit, so if you geocode a table with 100 rows, 100 credits will be wasted.
 
-You should take into account that the georeference tool that converts ZIP codes, IP addresses, counties, countries or cities doesn't count as geocoding but as georeferencing, so this way, you will be able to georeference your data in the map without using the credits.
+You should take into account that the georeference tool that convert ZIP codes, IP addresses, counties, countries, or cities doesn't count as geocoding but as georeferencing, so this way, you will be able to georeference your data in the map without using the credits.
```
Comment (iriberri) [inline faqs.md]: Isn't it "converts"? "the georeference tool"

**Comment by iriberri** [comment]
ping?

**Comment by csobier** [comment]
@brittanymicek 
This PR is closed, but but not merged.  Is there still a documentation requirement here or can I delete this branch from the Docs repo?

**Comment by andrewxhill** [comment]
i'd say not to delete it until we confirm the merge... just to be safe

On Thu, Aug 13, 2015 at 1:44 PM, csobier <notifications@github.com> wrote:

> @brittanymicek <https://github.com/brittanymicek>
> This PR is closed, but but not merged. Is there still a documentation
> requirement here or can I delete this branch from the Docs repo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/docs/pull/252#issuecomment-130776544>.
>



-- 

andrewxhill <http://twitter.com/andrewxhill>
http://cartodb.com

**Pull request merged**